src/RelayServer.ts

```diff
@@ -483,6 +483,23 @@ export class RelayServer extends EventEmitter {
         return maxPossibleGas;
     }
 
+    isSponsoredTx(req: RelayRequest | DeployRequest): boolean {
+        const { disableSponsoredTx, sponsoredDestinations } = this.config;
+
+        if (!disableSponsoredTx) {
+            return true;
+        }
+
+        if (
+            sponsoredDestinations &&
```

Comment:
If we make the property mandatory, we don't need it here.

Reply:
updated